themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
```

Comment:
I agree, I think that including more documentation with the code probably makes sense, especially given the discussions we've had about versioning documentation. I just want to be sure that if documentation in the code takes the place of documentation in the wiki, that we set up links accordingly so we don't have two versions of the same thing existing in parallel. This might be a good conversation for the wiki part of the next dev call.